src/track/taglib/trackmetadata_id3v2.cpp

```diff
@@ -148,23 +169,26 @@ TagLib::ID3v2::CommentsFrame* findFirstCommentsFrame(
     for (TagLib::ID3v2::FrameList::ConstIterator it(commentsFrames.begin());
             it != commentsFrames.end();
             ++it) {
```

Comment:
would it be possible to make this a range-based for loop too while you're at it?

Reply:
It was not only the range-based for loops that were missing: https://github.com/mixxxdj/mixxx/pull/4610/commits/55bc5ce8aab11d7f80688bc71c36f10204b7eb90

Now all downcasts use `downcastFrame()` instead of `dynamic_cast` consistently, accompanied by shared error reporting.

Reply:
Unfortunately some tests are failing now. Might need another force push.